app/services/idv/actions/verify_document_status_action.rb

```diff
@@ -5,6 +5,10 @@ def call
         process_async_state(async_state)
       end
 
+      def self.analytics_submitted_event
+        :idv_doc_auth_verify_document_status_submitted
+      end
```

Comment:
Our existing pattern for a lot of the Flow State Machine has been to define things as constants and dynamically read them.

I broke from that pattern because I believe class methods are a better choice, and more conventional for Ruby.

Reply:
Do you think we should revisit some of the others? Which are you referring to? (`STEP_INDICATOR_STEP`? others?)

Reply:
I mean even the  `FLOW_STATE_MACHINE_SETTINGS` constants count too in controllers.

I think we could file it as a nice-to-have that we switch to methods (maybe even add some syntax sugar/DSL-like methods to declare them), but our usages aren't complicate enough that it's a problem.

for example if we subclassed steps more, methods would be much better than constants because it's clearer to override a method than a constant.